test/utils.cpp

```diff
@@ -199,6 +252,26 @@ TEST_CASE("run_program()", "[utils]")
 	REQUIRE(utils::run_program(argv, "") == "hello world");
 }
 
+TEST_CASE("resolve_tilde() replaces ~ with the $HOME directory", "[utils]")
+{
+	SECTION("prefix-tilde replaced")
+	{
+		setenv("HOME","test",1);
```

Comment:
I didn't mention it in my review since @ThangaAyyanar already commented on it, but his comment is gone now, so: please use `TestHelpers::EnvVar` here (from test-helpers.h header). It saves the original value of the variable and restores it when the test completes (no matter its result), so this test won't affect others in any way.

Reply:
Looks like you forgot to remove this `setenv` when you added `envVar`.